here-naksha-app-service/src/test/java/com/here/naksha/app/service/NakshaAppTest.java

```diff
@@ -919,6 +925,89 @@ void tc0816_testGetByTileWithUnsupportedTileType() throws Exception {
     readFeaturesByTileTestHelper.tc0816_testGetByTileWithUnsupportedTileType();
   }
 
+  @Test
```

Comment:
Let's use searchTestHelper class. "someone" in past had raised a concern of this NakshaAppTest file getting bigger ;-) (and I still agree with that)

Reply:
> "someone" in past had raised a concern of this NakshaAppTest file getting bigger ;-)

Guilty as charged! But my main concern is not only the fact of the file getting bigger but also the need for @Order annotations and not splitting this one big suite into separate ones - I would very much like that. That's for another discussion though :D

I will move it to separate class for now as you suggest

Reply:
Pls also add a backlog ticket so we don't forget about it. Having a separate test class now, is already one step closer towards having split execution of individual classes.
Later, we may just need to avoid triggering from NakshaAppTest and will let all test classes run in parallel on their own. But I still don't think we will ever be able to prevent using @Order annotation (within individual test class), unless we choose to get into significant "setup" overhaead.
Nevertheless, I agree, it is discussion for another day! There could be an easier way when we look into it  with dedicated time.

Reply:
done